Modules/_sqlite/module.c

```diff
@@ -108,16 +108,16 @@ pysqlite_complete_statement_impl(PyObject *module, const char *statement)
 _sqlite3.register_adapter as pysqlite_register_adapter
 
     type: object(type='PyTypeObject *')
-    caster: object
+    callable as adapter: object
     /
 
-Registers an adapter with sqlite3's adapter registry.
+Register a function to adapt Python types to SQLite types.
```

Comment:
Maybe this is better:

```suggestion
Register a function to adapt Python objects to SQLite values.
```

Reply:
+1, I think this is better.